src/components/Book.js

```diff
@@ -15,28 +18,44 @@ const Book = () => {
 
   const booksArray = Object.values(books)[0];
   return (
-    <tbody className="table_body">
+    <>
       {booksArray.map((book) => (
-        <tr key={book.id}>
-          <td>
-            {book.title}
-            {' '}
-            by
-            {' '}
-            {book.author}
-          </td>
-          <td>
-            <button
-              type="button"
-              className="remove_btn"
-              onClick={() => handleRemoveBook(book.id)}
-            >
-              Remove
-            </button>
-          </td>
-        </tr>
+        <section className="each-book" key={book.id}>
+          <div className="bk-details">
+            <h5 className="entry-category">Category</h5>
+            <h3 className="entry-title">{book.title}</h3>
+            <h5 className="entry-author">{book.author}</h5>
+            <div className="bk-actions">
+              <button type="button" className="btn-type1">Comments</button>
+              <hr className="vertical line-1" />
+              <button
+                type="button"
+                className="remove_btn btn-type1"
+                onClick={() => handleRemoveBook(book.id)}
+              >
+                Remove
+              </button>
+              <hr className="vertical line-1" />
+              <button type="button" className="btn-type1">Edit</button>
+            </div>
+          </div>
+          <div className="progress-cont">
+            <CircularProgressbar value={66} />
+          </div>
+          <div>
+            <span className="percentage">64%</span>
+            <br />
```

Comment:
- [ ] Good job on the styling but please ensure that the space between the progress bar and percentage matches that of the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/222899914-0db0d7a8-bcad-44fa-bc7d-4b749d07508e.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/222899929-a3f280f6-79d5-4aa1-90e3-0d3347051c75.png)